### PR TITLE
Vrf 252 update event logs for o 11 y

### PR DESCRIPTION
**Pull request by jinhoonbang**

### CODE REVIEW

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go [hunk 1]
```diff
@@ -512,7 +512,7 @@ func (c *coordinator) filterUnfulfilledCallbacks(
 		aHeight := callbacksRequested[a].NextBeaconOutputHeight + callbacksRequested[a].ConfDelay.Uint64()
 		bHeight := callbacksRequested[b].NextBeaconOutputHeight + callbacksRequested[b].ConfDelay.Uint64()
 		if aHeight == bHeight {
-			return callbacksRequested[a].Callback.GasAllowance.Int64() < callbacksRequested[b].Callback.GasAllowance.Int64()
+			return callbacksRequested[a].GasAllowance < callbacksRequested[b].GasAllowance
```
Comment (vreff) [inline core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go:515]: Was the type a big int before?

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go [hunk 2]
```diff
@@ -521,11 +521,11 @@ func (c *coordinator) filterUnfulfilledCallbacks(
 		// Check if there is room left in the batch. If there is no room left, the coordinator
 		// will keep iterating, until it either finds a callback in a subsequent output height that
 		// can fit into the current batch or reaches the end of the sorted callbacks slice.
-		if c.coordinatorConfig.BatchGasLimit-currentBatchGasLimit < (r.Callback.GasAllowance.Int64() + c.coordinatorConfig.CallbackOverhead) {
+		if c.coordinatorConfig.BatchGasLimit-currentBatchGasLimit < (int64(r.GasAllowance) + c.coordinatorConfig.CallbackOverhead) {
```
Comment (vreff) [inline core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go:524]: nit: if this is now an int64 does it need type casting?

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go (hunk 1 above)
Comment (makramkd) [reply core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go:515]: Seems like it: https://github.com/smartcontractkit/chainlink/blob/d03d9772c62e648f54581f794837d13b15c27a74/core/gethwrappers/ocr2vrf/generated/vrf_coordinator/vrf_coordinator.go#L45-L54

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go (hunk 2 above)
Comment (makramkd) [reply core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go:524]: It seems like it's a `uint32` now. https://github.com/smartcontractkit/chainlink/pull/8049/files#diff-e3f18c3afd30cd2ce069d58e79d0a04f5cff87802e746b179c3437dd7bcb6f5eR1670

**Comment by vreff** [comment]
Approved. Looks like gasAllowance has always been a uint32, might have been erroneously changed prior to this PR which corrects it.

**Review by vreff — APPROVED** [review]

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go (hunk 2 above)
Comment (jinhoonbang) [reply core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go:524]: yeah it's a `uint32` in the `requestRandomnessFulfillment` [here](https://github.com/smartcontractkit/threshold-vrf/blob/master/contracts/VRFCoordinator.sol#L63). Seems like gasAllowance can be `uint32` everywhere.

**Pull request merged**